src/features/auth/components/LoginForm.tsx

```diff
@@ -54,20 +60,26 @@ export const LogInForm: FC<Props> = ({ onSubmit }) => {
   return (
     <Form data-testid='loginForm' onSubmit={handleSubmit(onSubmit)}>
       <Form.Group>
-        <Form.Label htmlFor='email'>Email</Form.Label>
-        <Form.Control id='email' type='email' {...register('email')} placeholder='Email' isInvalid={!!errors.email} />
+        <Form.Label htmlFor='email'>{t('email', { ns: 'common' })}</Form.Label>
+        <Form.Control
+          id='email'
+          type='email'
+          {...register('email')}
+          placeholder={t('emailPlaceholder', { ns: 'common' })!}
```

Comment:
Some props, particularly `placeholder` support argument type `string | undefined` whereas this conflicts with `t()` return type `string | null`. Since `t()` will always return a value, I used the non-null assertion operator `!` to inform typescript that the code is safe and valid. If a translation string is missing, `t()` will return the key name, so it always returns something.